tests/cypress/e2e/for-teacher_page/customize_class_page/customize_class.cy.js

```diff
@@ -1,22 +1,28 @@
-import {loginForTeacher} from '../../tools/login/login.js'
-import { createClass } from "../../tools/classes/class";
+import { loginForTeacher, loginForStudent } from '../../tools/login/login.js'
+import { ensureClass } from "../../tools/classes/class";
 
 describe('customize class page', () => {
+    let className;
     beforeEach(() => {
       loginForTeacher();
-      createClass();
+      className = ensureClass();
       cy.getBySel('view_class_link').first().click(); // Press on view class button
       cy.getBySel('customize_class_button').click(); // Press customize class button
+
+      // Remove any customizations that already exist to get the class into a predictable state
+      // This always throws up a modal dialog
+      cy.getBySel('remove_customizations_button').click({ force: true });
```

Comment:
This is actually pretty good! I think we can use this one instead of create class every single time?

Reply:
Yes, I was trying to save the database a little by not creating something new for every test run.

I've seen that kind of thing done at work and sooner or later the database always gets in trouble after you've run the test 1000s of times 🥸

Reply:
Cool! I'm refactoring the cypress tests and will take this into account!